mne/epochs.py

```diff
@@ -642,7 +647,7 @@ def plot_psd_topomap(self, bands=None, vmin=None, vmax=None, proj=False,
             self, bands=bands, vmin=vmin, vmax=vmax, proj=proj, n_fft=n_fft,
             ch_type=ch_type, n_overlap=n_overlap, layout=layout, cmap=cmap,
             agg_fun=agg_fun, dB=dB, n_jobs=n_jobs, normalize=normalize,
-            cbar_fmt=cbar_fmt, outlines=outlines, verbose=None)
+            cbar_fmt=cbar_fmt, outlines=outlines, verbose=None, show=show)
```

Comment:
... and if `show` doesn't come before `verbose` in `plot_psd_topomap`, please fix that, too.